src/oauth/index.js

```diff
@@ -3,10 +3,22 @@ createNameSpace("realityEditor.oauth");
 import { loadToken } from './tokens.js';
 
 (function(exports) {
+    function getToolboxEdgeBasePath() {
+        const windowPath = window.location.pathname;
+        return '/' + windowPath.split('/').slice(2,6).join('/') + '/i/whud7837yhd'; // => (/stable dropped when connecting to edge-server rather than ui) /n/networkId/s/networkSecret
```

Comment:
What is the 6 accomplishing here?

Reply:
Limiting the base path to `/n/networkId/s/networkSecret` and not anything further beyond that. I don't think it's actually necessary, but it was mostly general paranoia about accidentally having the relative path of the script included in the URL. I can change it to `.slice(2)` if you think that would be cleaner.

Reply:
I tend to be against any and all magic numbers but it's certainly a better solution than fully parsing the /n/{}/s/{} url structure to manually remake it. I'm okay with it